scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -1654,7 +1654,7 @@ function TTRPG_stats(faction, comp, mar, class = "marine", other_spawn_data={})
 			if (base_group == "astartes"){
 				ranged_hands_limit = 2
 			} else if base_group == "tech_priest" {
-				ranged_hands_limit = 1+(technology/100);;
+				ranged_hands_limit = 2+(technology/100); //Bro, let them be able to have at least some weapon.
```

Comment:
> i mean this makes them functionally better than our base marines, why give them weapons when people are essentially just using them as forge point sinks which is their main reasonable use

A good argument, with which I agree completely.

My problem is, that Tech Priests, according to wahapedia at least, should have Omnissian axes and Mechanicus pistols and overall pretty good stats. Also, they cost 2.5 times a single space marine model.

So, tldr: I wanted them to be good units, maybe even somewhat strong, while also have at least some weapon and not be over-encumbered with it (I'm not sure if they can wield any weapon at all rn, without going over the limit).
As such, my options were:
- Make them good all-round. Probably should've increased their price or something as well.
- Make special, light weapon items just for them, so they won't go over the limit.

I can tweak this stuff, as I just want to make them a) good and b) not over-encumbered by default.

Reply:
> The player probably dosn't need to have to augment tech priests it's outside of the cms purview so maybe the priest could come spawned with weaponry built into their body map?

Could work, but feels a bit lazy. Not super important part of the game, but that solution may feel kinda... shallow?
But that's just first impression.

> lightened weapons is a reasonable fix, maybe it's where using the artifact/mastercrafed system to lighten items could be useful?

Well, if I lighten default weapons with quality (the ones that marines can use as well), so that these guys can use it, the player will just take them off and use on marines.

Creating a separate item and forbidding equipping it on marines will work better, but again - feels a little bit gamey. Can argue that Mechanicus weapons have small handles and triggers and stuff, so yeah, could work.

But in general, I consider various choice locks to be something that should be used carefully, as to not break immersion. Why can't a marine use any weapon in the world? Gamey arbitrary locks are a bit... meh (if there is no good explanation).